conans/test/functional/generators/cmake_find_package_multi_test.py

```diff
@@ -408,3 +408,88 @@ def package_info(self):
 
         self.assertIn('set(requirement_LIBRARY_LIST_RELEASE lib_both lib_release)', content_release)
         self.assertIn('set(requirement_LIBRARY_LIST_DEBUG lib_both lib_debug)', content_debug)
+
+    def cpp_info_filenames_test(self):
+        client = TestClient()
+        client.run("new hello/1.0 -s")
+        indent = '\n        '
+        replace_in_file(
+            os.path.join(client.current_folder, "conanfile.py"),
+            search='self.cpp_info.libs = ["hello"]',
+            replace=indent.join([
+                'self.cpp_info.name = "hello_1"',
+                'self.cpp_info.filenames["cmake_find_package_multi"] = "hello_1"',
+                'self.cpp_info.names["cmake_find_package_multi"] = "MYHELLO"',
+                'self.cpp_info.components["1"].names["cmake_find_package_multi"] = "HELLO1"',
+                'self.cpp_info.components["1"].libs = [ "hello" ]'
+            ]),
+            output=client.out
+        )
+        client.run("create .")
+
+        client.run("new hello2/1.0 -s")
+        replace_in_file(
```

Comment:
There are many `replace_in_file()` that make the test difficult to read. I'd rather use a minimal conanfile here with the components or things that are needed to the test and avoid tying the test to the `conan new` generated files.